library/odoo

```diff
@@ -2,15 +2,15 @@ Maintainers: Christophe Monniez <moc@odoo.com> (@d-fence)
 GitRepo: https://github.com/odoo/docker
 GitCommit: ea74918b211215176bda85854a771be1c5c97871
 
-Tags: 17.0, 17, latest
+Tags: 17.0, 17, 17.0-20240522, 17-20240522, latest
 Architectures: amd64, arm64v8, ppc64le
 Directory: 17.0
 
-Tags: 16.0, 16
+Tags: 16.0, 16, 16.0-20240522, 16-20240522
```

Comment:
We don't rely on it often, but ordering the tags from most specific version to least specific version should minimize the diff here and in future PRs. We use the last tag in the list (the least specific) to help the diff minimize the "rename from -> rename to" output.
```suggestion
Tags:  16.0-20240522, 16-20240522, 16.0, 16
```